pandas/tests/io/parser/c_parser_only.py

```diff
@@ -205,6 +205,15 @@ def test_custom_lineterminator(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_crlf_lineterminator(self):
+        data = 'a,\n,c\r\n1,2,3\r\n4,\r,6'
```

Comment:
can you add the issue number. is this *only* c_parser? (I think yes), just confirming